amgut/lib/data_access/ag_data_access.py

```diff
@@ -1319,9 +1325,12 @@ def get_barcode_info_by_kit_id(self, ag_kit_id):
         return results
 
     def search_handout_kits(self, term):
-        sql = """select kit_id, password, barcode, verification_code
-                 from ag_handout_kits where kit_id like %s
-                 or barcode like %s"""
+        sql = """SELECT kit_id, password, barcode, verification_code
+                 FROM ag_handout_kits
+                 JOIN (SELECT kit_id, barcode, sample_barcode_file
+                    FROM ag.ag_handout_barcodes
+                    GROUP BY kit_id, barcode) AS hb USING (kit_id)
+                 WHERE kit_id LIKE %s or barcode LIKE %s"""
         cursor = self.get_cursor()
         liketerm = '%%' + term + '%%'
         cursor.execute(sql, [liketerm, liketerm])
```

Comment:
I think this file is unchanged from what I already reviewed in #428. Let me know if I'm mistaken and should review this here, too.